Mathlib/RingTheory/IntegralClosure/IsIntegralClosure/Basic.lean

```diff
@@ -356,6 +356,32 @@ theorem isIntegral_leadingCoeff_smul [Algebra R S] (h : aeval x p = 0) :
 
 end
 
+lemma quotient_isIntegralElem_of_monic (g : S[X]) (mon : g.Monic) (I : Ideal S[X]) (h : g ∈ I) :
```

Comment:
```suggestion
lemma quotient_isIntegralElem_of_monic {g : S[X]} (mon : g.Monic) {I : Ideal S[X])} (h : g ∈ I) :
```
Also, maybe this can be called `Polynomial.Monic.something` so that dot notation becomes available.